### PR TITLE
Remove LicenseResolver

### PR DESCRIPTION
**Pull request by mnonnenmacher**
This is the first part of a cleanup the removes code that duplicates functionality of the `LicenseInfoResolver`. See the commit messages for details.

The next PR will remove the `LicenseFindings` and `CopyrightFindings` classes.

### CODE REVIEW

File: model/src/test/kotlin/utils/LicenseResolverTest.kt
Comment (fviernau) [inline model/src/test/kotlin/utils/LicenseResolverTest.kt:171]: Do we have the tests still somewhere "duplicated" ?

Comment (mnonnenmacher) [reply model/src/test/kotlin/utils/LicenseResolverTest.kt:171]: Not exactly the same, but the `LicenseInfoResolver` is thoroughly tested.

**Comment by sschuberth** [comment]
This is looking good to me, but I'll leave it to @fviernau to approve as he's more into the topic.

File: model/src/test/kotlin/utils/LicenseResolverTest.kt
Comment (fviernau) [reply model/src/test/kotlin/utils/LicenseResolverTest.kt:171]: I was trying to find tests which cover the cases tested here, but to me it is not obvious whether `LicenseInfoResolverTest` accomplishes that. For that reason I believe it's not a good idea to delete these. But it might just be me because I wrote these (and found several bugs while writing them). So not blocking this one.

**Review by fviernau — APPROVED** [review]

**Pull request merged**
